public/jira/rollup/readme.md

```diff
@@ -0,0 +1,20 @@
+### `/rollup/rollup.js` Module
+
+The `/rollup/rollup.js` module is responsible for aggregating and rolling up data across parent-child relationships within a hierarchy of Jira issues or releases. This rollup process helps generate a hierarchical view of project data, which can be used for reporting purposes, showing the relationships between tasks, epics, and releases.
```

Comment:
As a high overview of this module, I think thats good enough.
Maybe for a next iteration we can go more into details about the main steps of the rollup flow like:

- Adding the reporting hierarchy
    -       grouping the issues/ releases by hierarchy level or type `groupIssuesByHierarchyLevelOrType`
    -       build parent-children relationship `addChildrenFromGroupedHierarchy`
- Performs the actual data aggregation based on the reporting hierarchy `rollupGroupedReportingHierarchy`
    - mention callback functions for creating metadata and rollup data.